### ISSUE TITLE
Is project abandoned/neglected?

### ISSUE THREAD

**Issue by FluorescentHallucinogen** [koush/Superuser#273]
@koush, Is the project abandoned/neglected?

**Comment by pjc123** [comment]
I must say that it worked flawlessly while I used it, but I could not wait anymore for a Lollipop update so switched to SuperSU.

**Comment by FluorescentHallucinogen** [comment]
SuperSU is closed source. Both, root permission manager app and su binary should be open source. It's the gateway to root on your device. It must be open for independent security analysis. Obscurity (closed source) is not security. BTW, free version of SuperSU has no some features from paid SuperSU Pro version.

The only active development fork of Superuser is [CyanogenMod/Superuser](https://github.com/CyanogenMod/Superuser). But Superuser app has been excluded from CyanogenMod (http://review.cyanogenmod.org/83759). Its functions relegated to Privacy Guard. It seems that the development of Superuser will be stopped.

This means that at this moment there is no working open source root permission manager app and open source su binary for Android >=5.0 ROMs (AOSP based, not CyanogenMod based).

**Comment by jeekajoo** [comment]
there is also this fork: https://github.com/SlimRoms/Superuser
I have the same concerns as you: http://forum.xda-developers.com/general/help/lollipop-superuser-fork-opensource-t3022824

**Comment by unforgettableid** [comment]
[koush/Superuser](https://github.com/koush/Superuser) saw its most recent commit in Nov. of '14.  It seems to be no longer actively maintained.

Excluding automatic translation checkins, [CyanogenMod/Superuser](https://github.com/CyanogenMod/Superuser) saw its last commit in Dec. of '14.  It also is probably no longer actively maintained.

The main fork seeing active development is [phhusson/Superuser](https://github.com/phhusson/Superuser), which saw its last commit fifteen days ago.  See also #268 and #283.

**Comment by jeekajoo** [comment]
Yeah, I already knew that @phhusson was working on a fork (https://github.com/SlimRoms/Superuser/issues/4#issuecomment-125494463) but thanks for the heads up.
